### PR TITLE
Mekyle.poweredby

### PR DESCRIPTION
**Pull request by SpaceGoat97**
Added a "Powered by commonwealth" button on the bottom of custom domains that link people to commonwealth.im

### CODE REVIEW

File: client/scripts/views/components/sidebar/index.ts [hunk 1]
```diff
@@ -594,8 +594,18 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
         m('br'),
         app.isLoggedIn() && (app.chain || app.community) && m(SubscriptionButton),
         app.chain && m(ChainStatusModule),
-      ])
-    ];
+        app.isCustomDomain && m(SidebarQuickSwitcher),
+        m('br'),
+        m('br'),
+      ]),
+      app.isCustomDomain(),
```
Comment (jnaviask) [inline client/scripts/views/components/sidebar/index.ts:601]: Why is there a hanging `isCustomDomain` here? Not a mithril component -- needs to be `&&` with the PoweredBy link below.

File: client/scripts/views/components/sidebar/index.ts [hunk 2]
```diff
@@ -594,8 +594,18 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
         m('br'),
         app.isLoggedIn() && (app.chain || app.community) && m(SubscriptionButton),
         app.chain && m(ChainStatusModule),
-      ])
-    ];
+        app.isCustomDomain && m(SidebarQuickSwitcher),
+        m('br'),
```
Comment (jnaviask) [inline client/scripts/views/components/sidebar/index.ts:598]: Do we always want these two line breaks?

File: client/styles/components/sidebar/index.scss [hunk 3]
```diff
@@ -308,6 +309,31 @@
...
+    position: absolute;
+    border-radius: 8px;
+    border-color: #F9F9F9;
+    font-size: 14px;
+    background-image: url("/static/img/powered-by.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-top: 10%;
+}
+.PoweredBy:hover{
```
Comment (jnaviask) [inline client/styles/components/sidebar/index.scss:324]: Why not combine this with the above class since the content is exactly the same?

**Review by jnaviask — CHANGES_REQUESTED** [review]
Few comments.

File: client/scripts/views/components/sidebar/index.ts (hunk 1 above)
Comment (SpaceGoat97) [reply client/scripts/views/components/sidebar/index.ts]: Updated and the extra line break is removed

**Review by jnaviask — COMMENTED** [review]
Quick Q before I QA.

File: client/scripts/views/components/sidebar/index.ts [hunk 4]
```diff
@@ -594,8 +594,16 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
         m('br'),
         app.isLoggedIn() && (app.chain || app.community) && m(SubscriptionButton),
         app.chain && m(ChainStatusModule),
-      ])
-    ];
+        app.isCustomDomain && m(SidebarQuickSwitcher),
```
[the remark this answers is not in the document]
Comment (SpaceGoat97) [reply client/scripts/views/components/sidebar/index.ts:597]: Ahhh thank you!, I forgot I added that there for some reason.

**Comment by jnaviask** [comment]
Any idea why it wiggles when I hover over it? Otherwise, good to merge.

File: client/scripts/views/components/sidebar/index.ts [hunk 5]
```diff
@@ -595,7 +595,6 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
         app.isLoggedIn() && (app.chain || app.community) && m(SubscriptionButton),
         app.chain && m(ChainStatusModule),
       ]),
-      app.isCustomDomain() &&
```
Comment (jnaviask) [inline client/scripts/views/components/sidebar/index.ts]: This means the link will display on all pages, which isn't what we want.

**Pull request merged**